server/shared/src/test/scala/org/http4s/server/middleware/TimeoutSuite.scala

```diff
@@ -47,7 +49,7 @@ class TimeoutSuite extends Http4sSuite {
   private val uncancelableReq = Request[IO](GET, uri"/uncancelable")
 
   def checkStatus(resp: IO[Response[IO]], status: Status): IO[Unit] =
-    resp.map(_.status).timeout(1.seconds).assertEquals(status)
+    resp.map(_.status).assertEquals(status)
```

Comment:
Why can the `timeout` go away here, I guess it was just acting as a test suite timeout with no real purpose in the tests?

Reply:
Precisely - the timeout was a spurious time control over the execution of the test, rather than the actual timeout that would come from the middleware.